python-sdk/src/astro/databricks/load_file/load_file_job.py

```diff
@@ -86,9 +95,40 @@ def _create_load_file_pyspark_file(
     return dbfs_file_path
 
 
-def _load_load_file_to_dbfs(api_client: ApiClient, input_file: File):
+def _load_local_file_to_dbfs(api_client: ApiClient, input_file: File):
     file_path = Path(input_file.path)
     dbfs_file_path = load_file_to_dbfs(
         local_file_path=Path(input_file.path), file_name=file_path.name, api_client=api_client
     )
     return dbfs_file_path
+
+
+def _load_secrets_to_databricks(api_client: ApiClient, input_file: File, secret_scope_name: str):
+    """
+    Load secrets into a secret scope in databricks. This can be used as a utility for loading
+    necessary secrets for accessing external data stores.
+
+    :param api_client: Databricks API client loaded with credentials
+    :param input_file: file for loading connection info
+    :param secret_scope_name: name of the secret scope to load info into
+    """
+    create_secrets(
+        scope_name=secret_scope_name,
+        filesystem_secrets=input_file.location.databricks_settings(),
+        api_client=api_client,
+    )
+
+
+def _find_file_type(input_file: File) -> str:
+    """
+    Since COPY INTO does not automatically detect schema, we need to ensure that the file type is
+    either set by the user, or inferable in the URL
+    :param input_file: file to parse
+    :return: file type of the uploaded file
+    """
+    if input_file.filetype:
+        return str(input_file.filetype)
+    if not input_file.filetype:
+        if "." in input_file.path:
```

Comment:
Perhaps we could use pathlib to handle this: `Path(input_file.path).suffix`

Reply:
@tatiana if there is no suffix and the user does not specify the file type I don't think there's much that we can do besides raise an error. Unless we want to try to list files and see if any of those files have a suffix?